src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/ViewExecutor.cs

```diff
@@ -104,7 +106,7 @@ public class ViewExecutor
             IView view,
             ViewDataDictionary viewData,
             ITempDataDictionary tempData,
-            MediaTypeHeaderValue contentType,
+            MediaTypeHeaderValue actionResultContentType,
```

Comment:
Not sure I buy this name is any better. The new name provides unnecessary information about the context in which `ExecuteAsync()` is called but says nothing more about how it's used. All this method should infer about the context is that the `Response` and its headers can be changed.